Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -945,6 +945,7 @@ def headers_for_type(type):
         'WebCore::WheelScrollGestureState': ['<WebCore/PlatformWheelEvent.h>'],
         'WebCore::WillContinueLoading': ['<WebCore/FrameLoaderTypes.h>'],
         'WebCore::WillInternallyHandleFailure': ['<WebCore/FrameLoaderTypes.h>'],
+        'WebCore::WindowProxyProperty': ['<WebCore/FrameLoaderTypes.h>'],
```

Comment:
Do we really need this? Can't we use `header: <WebCore/FrameLoaderTypes.h>`
and `[CustomHeader]` in the serialization.in instead?